CMakeLists.txt

```diff
@@ -689,4 +689,4 @@ if(UNIX AND NOT APPLE)
 		add_custom_target(uninstall
 			COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
 	endif()
-endif()
+endif()
```

Comment:
As well as remove this change please :3

Reply:
i copied the same CMakeLists.txt from the repo and still giving that for some reason

Reply:
Just unstage that change from you commit, i.e. amend the commit and don't commit the CMakeList.txt.